manager/assets/modext/core/modx.layout.js

```diff
@@ -287,6 +288,19 @@ Ext.extend(MODx.Layout, Ext.Viewport, {
                                 }
                             }
                         );
+
+                        var header = Ext.get('modx-leftbar-header');
+                        if (header) {
+                            var html = '';
+                            if (MODx.config.manager_logo !== '' && MODx.config.manager_logo !== undefined) {
+                                html += '<img src="' + MODx.config.manager_logo + '">';
+                            }
+                            html += '<span>' + MODx.config.site_name + '</span>';
+                            header.dom.innerHTML = html;
+                            header.on('click', function() {
```

Comment:
Can we just make this a plain old link so it's accessible and things like ctrl+click work out of the box?